templates/contribute.html

```diff
@@ -0,0 +1,109 @@
+
+                        <p>Don't forget to check our <a href="http://coach.djangogirls.org/">Coach Manual</a>!</p>
+                    </div>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-md-6">
+                    <h3>Improve the tutorial!</h3>
+                    <div class="cover">
+                        <a href="https://github.com/DjangoGirls/tutorial">
+                            <img src="{% static 'img/global/contribute/tutoedit.jpg' %}" alt="A note book and a pen." />
+                        </a>
+                    </div>
+                    <div class="info">
+                        <p>Something in the tutorial is not clear or there is a typo? You can improve the tutorial by making pull request on our <a href="https://github.com/DjangoGirls/tutorial">Github repository</a>.</p>
+
+                        <p>You can also work on the <a href="https://crowdin.com/project/django-girls-tutorial">translation</a> of the tutorial. If you want to join a team or launch a new translation, <a href="mailto:translations@djangogirls.org">contact us</a>.</p>
+                    </div>
+                </div>
+                <div class="col-md-6">
+                    <h3>Work on the website!</h3>
+                    <div class="cover">
+                        <a href="https://github.com/DjangoGirls/djangogirls">
+                            <img src="{% static 'img/global/contribute/debug.jpg' %}" alt="Computer screen with a console and a webpage." />
+                        </a>
+                    </div>
+                    <div class="info">
+                        <p>There is always something to do to make our website better. You can report bugs and typo, fix those or work on new features.</p>
+
+                        <p>We are waiting for your <a href="https://github.com/DjangoGirls/djangogirls"> pull requests!</a></p>
+                    </div>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-md-6">
+                    <h3>Help on Gitter!</h3>
+                    <div class="cover">
+                        <a href="https://gitter.im/DjangoGirls/tutorial">
+                            <img src="{% static 'img/global/contribute/gitter.jpg' %}" alt="A web page opened on a computer." />
+                        </a>
+                    </div>
+                    <div class="info">
+                        <p>Many people are doing our tutorial outside of a workshop. Sometimes, they get stuck and need help. How about spending a few hours in a week to help them?</p>
+
+                        <p>This <a href="https://gitter.im/DjangoGirls/tutorial">Gitter room </a> is designed to be a friendly space, especially to beginner. Remember that our <a href="https://djangogirls.org/pages/coc/">Code of Conduct</a> also apply there.</p>
```

Comment:
"... especially for beginners. Remember that our Code of Conduct also applies there."